app/src/main/res/values/strings.xml

```diff
@@ -243,10 +243,14 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_cycleway_title2">"Is there a cycleway here? What type?"</string>
     <string name="quest_cycleway_resurvey_title">"Is this still the cycleway situation here?"</string>
     <string name="quest_serviceType_title">"What kind of service road is this?"</string>
-    <string name="quest_streetLanes_title">"How many lanes does this road have?"</string>
-    <string name="quest_streetLanes_description">"If there are no road markings, it has no lanes."</string>
-    <string name="quest_streetLanes_answer_noLanes">"It has no marked lanes"</string>
-    <string name="quest_streetLanes_noLanes_estWidth">"Please estimate how many cars would fit alongside each other on this road."</string>
+    <string name="quest_lanes_title">"How many lanes for cars does this road have?"</string>
+    <string name="quest_lanes_answer_lanes">"Marked lanes"</string>
+    <string name="quest_lanes_answer_lanes_description">"Specify the total number of usable marked lanes (both sides). So, lanes occupied with parking cars don't count."</string>
```

Comment:
Maybe "lanes occupied with parking cars" -> "lanes for parking cars"

"occupied" seems possible to be misinterpreted as "occupied right now"? "for" seems shorter and also correct and without this issue

Not sure how to handle edge case of lanes available for driving for part of the day and as parking during evening and night.

Reply:
I don't want it to sound like that the user has to look for whether parking is allowed here or not but just look at the de-facto state of the road. If a side of a road is used for parking, even when everyone went to work, there will be one or another car left to denote that cars park here I think.

What about "So, lanes used by parking cars don't count"?

Reply:
I think I'll leave it as it is for now, because
- "lanes for parking cars" slightly suggests that user ought to look for official markings or signs
- "lanes used by parking cars" sounds not as nice as "occupied with" IMO and "occupied" also suggests (and I think it ought) that if there are a few parking cars scattered around (which act like traffic calming) that it is doesn't mean that the lane should not count at all